assets/stylesheets/sections/_menus.scss

```diff
@@ -542,7 +542,7 @@
 	// If done properly, might be an alternative for mobile.
 	// Seems not working properly with -webkit-perspective turned on.
 	/*@include responsive( mobile ) {
-		z-index: 190; // Higher than masterbar?
+		z-index: ; // Add an entry to _z-index.scss
```

Comment:
Can we just removed this commented block? Commented blocks of code should be avoided at all cost (history is always there if we need to refer back to it).